browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -2065,6 +2065,61 @@ L.CanvasTileLayer = L.Layer.extend({
 		}
 	},
 
+    _highlightColAndRow: function (textMsg, part) {
+        this._resetReferencesMarks();
+        var strTwips = textMsg.match(/\d+/g);
+        var references = [];
+        this._referencesAll = [];
+        var rectangles = [];
+        var strColor = 'ccffff';
```

Comment:
would be good to get css value from variable so we can theme it, @pedropintosilva can help which one to use
example: https://github.com/CollaboraOnline/online/blob/e19aa5c9c3f944cf6b3a21c03bbff183d6a1242f/browser/src/map/handler/Map.Feedback.js#L101

Reply:
Done